react/fields/Dropdown/Dropdown.js

```diff
@@ -178,7 +137,7 @@ export default class Dropdown extends Component {
 
     return (
       <div className={classNames}>
-        {this.renderLabel()}
+        <FieldLabel {...this.props} />
```

Comment:
The idea behind this is that the prop validation (special rules around label, id & labelProps, etc) can live inside FieldLabel. Happy to look at alternatives if this is still possible?